transport/src/commonTest/kotlin/com/genesys/cloud/messenger/transport/network/WebMessagingApiTest.kt

```diff
@@ -463,6 +489,22 @@ class WebMessagingApiTest {
             assertIs<Exception>(throwable)
         }
     }
+
+    @Test
+    fun `when getMessages receives 502 server error then retries with exponential backoff`() {
+        val configuration = TestValues.configuration
+        subject = buildWebMessagingApiWith(configuration) { retryEngine() }
+
+        assertTrue(true)
```

Comment:
This test does not have any value. assertTrue on hardcoded true will always pass.